test/IntegrationTests/Helpers/TestHelper.cs

```diff
@@ -90,7 +90,11 @@ public async Task<Container> StartContainerAsync(TestSettings testSettings, int
             Output.WriteLine($"Otlp Endpoint: {agentBaseUrl}");
             builder = builder.WithEnvironment("OTEL_EXPORTER_OTLP_ENDPOINT", agentBaseUrl);
             builder = builder.WithEnvironment("OTEL_METRIC_EXPORT_INTERVAL", "1000");
-            builder = builder.WithEnvironment("OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS", "AspNet");
+        }
+
+        foreach (var env in EnvironmentHelper.CustomEnvironmentVariables)
```

Comment:
it's `CustomEnvironmentVariables`, the ones what ever you add extra for specific tests.

Reply:
this comment is not about this change but the fact that `TestHelper.StartContainerAsync ` is used only [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/e013026f8bea18de9e143b1ad3d2ccb2faba41bb/test/IntegrationTests/AspNetTests.cs#L55) so probably it could be moved to `AspNetTests.cs`

Reply:
I do not know how often we will want to run a test app in a container. It seems like an edge case to me. I think that it makes sense to move a lot of that logic for now into `AspNetTests.cs`. Doing so will make it easier for newcomers to know that structuring a test app in that manner is for special circumstances only. I think that this can be done in a separate PR.